test/steps/coworker_steps.go

```diff
@@ -0,0 +1,33 @@
+package steps
+
+import (
+	"fmt"
+
+	"github.com/cucumber/godog"
+)
+
+// CoworkerSteps defines Gherkin step implementations around a coworker.
+func CoworkerSteps(suite *godog.Suite, fs *FeatureState) {
+	suite.Step(`^my coworker fetches updates$`, func() error {
+		_, err := fs.activeScenarioState.gitEnvironment.CoworkerRepo.Shell.Run("git", "fetch")
```

Comment:
We got tested and typesafe methods for such Git operations now. 🙂 

```go
_, err := fs.activeScenarioState.gitEnvironment.CoworkerRepo.Fetch()

Reply:
Nice, will update